datarepo-clienttests/src/main/java/testscripts/BulkLoadScale.java

```diff
@@ -0,0 +1,56 @@
+package testscripts;
+
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.*;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import utils.BulkLoadUtils;
+import utils.KubernetesClientUtils;
+
+public class BulkLoadScale extends runner.TestScript {
+  private static final Logger logger = LoggerFactory.getLogger(BulkLoadScale.class);
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public BulkLoadScale() {
+    super();
+    manipulatesKubernetes = true; // this test script manipulates Kubernetes
```

Comment:
Is that because it has a kubernetes section in the test config?

Reply:
Right - Since we're manipulating Kubernetes as a part of our test, the server config must have skipKubernetes = false, or the test runner will throw an error. This handles the case that we're running this locally, but it wouldn't actually test with the desired setup.